src/bentoml/__init__.py

```diff
@@ -58,6 +58,7 @@
     from . import catboost
     from . import detectron
     from . import diffusers
+    from . import diffusers_simple
```

Comment:
Should we encapsulate this to `bentoml.onediffusion`?

Reply:
slightly different, because diffusers_simple does not make many assumption like OneDiffusion.

Reply:
sg